### DISCUSSION TITLE
how to upgrade complex docker versions

### DISCUSSION THREAD

**Discussion by mikethecalamity** [renovatebot/renovate#29854] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab

### Please tell us more about your question or problem

One of the images from Docker hub I utilize is `bitnami/postgresql`, which has versions in a format like `14.10.0-debian-11-r23`. But when I upgrade the minor versions, the version suggested by Renovate is `14.11.0-debian-11-r17` instead of the latest `14.12.0-debian-12-r13`. I'm guessing it's due to the Debian 11 vs. 12. But how can I ignore this and take the latest regardless of the Debian version?

### Logs (if relevant)

<details><summary>Logs</summary>

```json
{
    "depName": "cdn.harbor.global.lmco.com/ext.hub.docker.com/bitnami/postgresql-repmgr",
    "currentValue": "14.10.0-debian-11-r17",
    "datasource": "docker",
    "replaceString": "14.10.0-debian-11-r17",
    "versioning": "docker",
    "autoReplaceStringTemplate": "{{newValue}}{{#if newDigest}}@{{newDigest}}{{/if}}",
    "updates": [
        {
            "bucket": "non-major",
            "newVersion": "14.11.0",
            "newValue": "14.11.0-debian-11-r17",
            "newMajor": 14,
            "newMinor": 11,
            "updateType": "minor",
            "branchName": "renovate-postgresql"
        },
        {
            "bucket": "major",
            "newVersion": "16.2.0",
            "newValue": "16.2.0-debian-11-r17",
            "newMajor": 16,
            "newMinor": 2,
            "updateType": "major",
            "branchName": "renovate-major-postgresql"
        }
    ],
    "packageName": "cdn.harbor.global.lmco.com/ext.hub.docker.com/bitnami/postgresql-repmgr",
    "warnings": [],
    "sourceUrl": "https://github.com/bitnami/containers",
    "registryUrl": "https://cdn.harbor.global.lmco.com",
    "sourceDirectory": "bitnami/postgresql-repmgr",
    "lookupName": "ext.hub.docker.com/bitnami/postgresql-repmgr",
    "currentVersion": "14.10.0",
    "isSingleVersion": true,
    "fixedVersion": "14.10.0-debian-11-r17"
}
```

</details>

**Comment by rarkins** [comment]
Try https://github.com/search?q=repo%3Arenovatebot%2Frenovate+bitnami&type=discussions first
